<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorForm</class>
 <widget class="QWidget" name="AuthorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Author Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="firstNameLabel">
     <property name="text">
      <string>First Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="firstNameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lastNameLabel">
     <property name="text">
      <string>Last Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lastNameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="descriptionLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="critiqueLabel">
     <property name="text">
      <string>Critique:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="critiqueLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ratingLabel">
     <property name="text">
      <string>Rating:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="pictureLabel">
     <property name="text">
      <string>Picture:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QWidget" name="pictureWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="pictureLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="picturePushButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="themesLabel">
     <property name="text">
      <string>Themes:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QListWidget" name="themesListWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="savePushButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="RatingWidget" name="ratingWidget">
     <property name="max">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RatingWidget</class>
   <extends>QWidget</extends>
   <header>RatingWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>firstNameLineEdit</tabstop>
  <tabstop>lastNameLineEdit</tabstop>
  <tabstop>descriptionLineEdit</tabstop>
  <tabstop>critiqueLineEdit</tabstop>
  <tabstop>pictureLineEdit</tabstop>
  <tabstop>picturePushButton</tabstop>
  <tabstop>themesListWidget</tabstop>
  <tabstop>cancelPushButton</tabstop>
  <tabstop>savePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
